pkg/queue/cluster_queue_best_effort_fifo_test.go

```diff
@@ -125,6 +136,26 @@ func TestClusterQueueBestEffortFIFO(t *testing.T) {
 				cq.QueueInadmissibleWorkloads()
 			}
 
+			if test.deleteQueue != nil {
```

Comment:
This looks error prone, can we create a simpler separate test like this: https://github.com/kubernetes-sigs/kueue/blob/664c6b9bae3aa6e5d3d5b048298418a6080ed99f/pkg/queue/cluster_queue_impl_test.go#L140